### PR TITLE
Added appveyor config.

### PR DESCRIPTION
**Pull request by brian-reichle**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+skip_branch_with_pr: true
+os: Visual Studio 2019
+
+configuration:
+  - Debug
+  - Release
+
+environment:
+  WTG_ANALYZERS_VERSION: 3.3.0
```
Comment (yaakov-h) [inline .appveyor.yml]: Do we still need this? At best we should probably rename it.

File: .appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+version: 3.3.0.{build}
```
Comment (yaakov-h) [inline .appveyor.yml:1]: Should we take this down a notch?

File: .appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  - Release
+
+environment:
+  WTG_ANALYZERS_VERSION: 3.3.0
+
+init:
+  - git config --global core.autocrlf true
+
+before_build:
+  - dotnet restore src
```
Comment (yaakov-h) [inline .appveyor.yml:14]: I didn't know you could specify a whole folder like that!

File: .appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+build:
+  parallel: true
+  project: src\BulkAnalysisRunner.sln
+  verbosity: minimal
```
Comment (yaakov-h) [inline .appveyor.yml:19]: Should we also zip up artifacts?

File: .appveyor.yml (hunk 1 above)
Comment (brian-reichle) [reply .appveyor.yml]: Good point, I'll remove it.

File: .appveyor.yml (hunk 2 above)
Comment (brian-reichle) [reply .appveyor.yml:1]: It was set to match the [version in Directory.Build.props](https://github.com/WiseTechGlobal/BulkAnalysisRunner/blob/a07ad55e7ea55f9f5b94acd851ad61daa679b3c9/Directory.Build.props#L12) which was in turn set to match the Roslyn version.

File: .appveyor.yml (hunk 3 above)
Comment (brian-reichle) [reply .appveyor.yml:14]: If you don't specify a solution/project, it looks for one in the current directory. If you specify a directory, it performs that search in the provided directory.

File: .appveyor.yml (hunk 4 above)
Comment (brian-reichle) [reply .appveyor.yml:19]: Probably not much point unless we have somewhere to send it.

**Review by yaakov-h — APPROVED** [review]

**Pull request merged**